storage/mysql/log_storage.go

```diff
@@ -259,15 +261,26 @@ func (m *mySQLLogStorage) ReadWriteTransaction(ctx context.Context, tree *trilli
 }
 
 func (m *mySQLLogStorage) AddSequencedLeaves(ctx context.Context, tree *trillian.Tree, leaves []*trillian.LogLeaf) ([]*trillian.QueuedLogLeaf, error) {
```

Comment:
Is this func (on LogStorage impl) needed, or is this a `LogTreeTX` thing?

Reply:
It is both, similarly to `QueueLeaves`. Currently both are transactional, but we discussed earlier that we wanted to allow the `LogStorage` ones treat entries independently. WDYT?

Reply:
SGTM